cmd/spec/example_pubkey.go

```diff
@@ -29,4 +32,11 @@ var PubkeyListResponse = payloads.PubkeyListResponse{
 			FingerprintLegacy: "ee:f1:d4:62:99:ab:17:d9:3b:00:66:62:32:b2:55:9e",
 		},
 	},
+	Metadata: page.Metadata{
+		Total: 1,
+	},
+	Links: page.Links{
+		Previous: "/api/provisioning/v1/pubkeys?limit=100&offset=0",
```

Comment:
Here it should be empty as well, right?

Reply:
for the pubkey, you'd also need another key, as currently we have only one key in the result... 
But here, you can do a trick or rather demonstrate a feature and do `Total = 3`, that way it will make sense :)

Reply:
yes, so just s sources the total will be four and pubkey will be total of 3

Reply:
Yup :)